server/views/components/cookie-notification/cookie-notification.njk

```diff
@@ -1,12 +1,13 @@
-<div id="cookie-notification" class="cookie-notification {{ {s:'HNM6', m:'HNM4'} | fontClasses }}">
+<div class="cookie-notification is-hidden {{ {s:'HNM6', m:'HNM4'} | fontClasses }} js-info-banner"
+  data-cookie-name="cookies_accepted">
```

Comment:
I love underscores, but we don't use them anywhere else.

Reply:
We've also created the namespace `WC_` - but maybe that's for server cookies?
I'd like to keep it as it might make the legal docs easier to maintain.